ANMN/LTSP/TSaggregator/TSagg_metadata.json

```diff
@@ -104,26 +102,26 @@
             },
         "TEMP": {
             "ancillary_variables": "TEMP_quality_control",
-            "_FillValue": 999999.0, 
+            "_FillValue": 999999.0,
             "long_name": "sea_water_temperature",
-            "standard_name": "sea_water_temperature", 
+            "standard_name": "sea_water_temperature",
             "units": "degrees_Celsius",
-            "valid_max": 40.0, 
+            "valid_max": 40.0,
             "valid_min": -2.5
             },
         "TEMP_quality_control": {
-            "_Unsigned": "false", 
-            "_FillValue": 99, 
+            "_Unsigned": "false",
+            "_FillValue": 99,
             "flag_meanings": "No_QC_performed Good_data Probably_good_data Bad_data_that_are_potentially_correctable Bad_data Value_changed Not_used Not_used Not_used Missing_value",
-            "flag_values": "0, 1, 2, 3, 4, 5, 6, 7, 8, 9", 
-            "long_name": "quality flag for sea_water_temperature", 
+            "flag_values": "0, 1, 2, 3, 4, 5, 6, 7, 8, 9",
+            "long_name": "quality flag for sea_water_temperature",
             "quality_control_conventions": "IMOS standard flags",
             "quality_control_global": "A",
             "quality_control_global_conventions": "Argo reference table 2a (see http://www.cmar.csiro.au/argo/dmqc/user_doc/QC_flags.html), applied on data in position only (between global attributes time_deployment_start and time_deployment_end)",
```

Comment:
Remove **all** 'quality_control_global' and 'quality_control_global_conventions' attributes, as we won't be using these in the aggregated files.

Reply:
removed.